lib/phlex/rails/helpers/link_to.rb

```diff
@@ -3,7 +3,21 @@
 module Phlex::Rails::Helpers::LinkTo
 	extend Phlex::Rails::HelperMacros
 
-	# @!method link_to(...)
-	# 	@return [nil]
-	register_output_helper :link_to
+	def link_to(*args, **kwargs, &block)
```

Comment:
It is probably worth making a macro for this, since there are a lot of helpers that take an `id` as a keyword argument. A few other helpers, such as `turbo_frame_tag` take an `id` as a positional argument.